editor/script/editor.js

```diff
@@ -797,12 +797,11 @@ var defaultPanelPrefs = {
 
 function getPanelPrefs() {
 	// (TODO: weird that engine version and editor version are the same??)
-	var useDefaultPrefs = ( localStorage.engine_version == null ) ||
-							( localStorage.panel_prefs == null ) ||
-							( JSON.parse(localStorage.engine_version).major < 6 ) ||
-							( JSON.parse(localStorage.engine_version).minor < 0 );
-
-	var prefs = useDefaultPrefs ? defaultPanelPrefs : JSON.parse( localStorage.panel_prefs );
+	var storedEngineVersion = Store.get('engine_version');
+	var useDefaultPrefs = (!storedEngineVersion) ||
+	                      (storedEngineVersion.major < 6) ||
+	                      (storedEngineVersion.minor < 0);
```

Comment:
kept the logic here the same as before, but is this correct? when would the minor version be < 0?

Reply:
it wouldn't ever be! I just update this test whenever I make a change that would break the stored preferences and requires them to be reset, so for example if version 6.2 had required that, I would have updated the minor version test

Reply:
ah ok; now that the storage has been separated to its own module, it'd probably be a good idea in the future to make a "migrations" setting to encapsulate logic like this in one place